lib/jsDocToJson.js

```diff
@@ -0,0 +1,21 @@
+const exec = require('child_process').exec;
```

Comment:
Is there a specific reason why this has been extracted to a separate file? Can we instead have `jsDocTJson` inside `documentation.js` that way it would be easy to use it later for example.

```
let doc = require('documentation');
doc.jsDocToJson(...);
```
What do you think?

Reply:
I already tried as you suggest, but I encountered a problem that made me split the files.
Maybe you have a better way to address it.

 The problem is that `documentation.js` contains also the following code at the top level:
```javascript
if (!process.env.TAIKO_SKIP_DOCUMENTATION) {
  const sourceFiles = 'lib/taiko.js lib/elementWrapper/*.js';
  const outputFile = 'lib/api.json';
  jsDocToJson(sourceFiles, outputFile);
}
```

So when I do `let doc = require('documentation');` in the file `test/docs-tests/generateWebsite.js`, the code above is immediately run.
The net result is that `jsDocToJson` is executed twice 😮

Splitting the files allowed me to avoid importing the top-level code of `documentation.js`. Do you have in mind any other option to fix that?

Reply:
Ah ok, now I see. You can try this

```
if (require.main === module && !process.env.TAIKO_SKIP_DOCUMENTATION) {
   const sourceFiles = 'lib/taiko.js lib/elementWrapper/*.js';
   const outputFile = 'lib/api.json';
   jsDocToJson(sourceFiles, outputFile);
 }
```

Reply:
Cool!

I did not know. I keep learning a lot, thank you! 😀